src/main/java/seedu/address/ui/EntityCard.java

```diff
@@ -79,6 +90,8 @@ public EntityCard(Entity entity, int displayedIndex) {
      * @param entity type of Entity.
      */
     private void generateParticipantCard(Entity entity) {
+        this.idIcon.setImage(getImage("/images/student_icon.png"));
```

Comment:
Maybe store paths to images as constants?

Reply:
ah ok!